internal/tasks/manifests_test.go

```diff
@@ -595,11 +595,11 @@ func TestCheckVulnerabilitiesForNextManifest(t *testing.T) {
 	//to Clair for indexing, but since Clair is not done indexing yet, images
 	//stay in vulnerability status "Pending" for now
 	s.Clock.StepBy(30 * time.Minute)
-	expectSuccess(t, j.CheckVulnerabilitiesForNextManifest()) //once for each manifest
-	expectSuccess(t, j.CheckVulnerabilitiesForNextManifest())
-	expectSuccess(t, j.CheckVulnerabilitiesForNextManifest())
-	expectSuccess(t, j.CheckVulnerabilitiesForNextManifest())
-	expectSuccess(t, j.CheckVulnerabilitiesForNextManifest())
+	expectSuccess(t, j.CheckVulnerabilitiesForNextPendingManifest()) //once for each manifest
+	expectSuccess(t, j.CheckVulnerabilitiesForNextPendingManifest())
+	expectSuccess(t, j.CheckVulnerabilitiesForNextPendingManifest())
+	expectSuccess(t, j.CheckVulnerabilitiesForNextPendingManifest())
+	expectSuccess(t, j.CheckVulnerabilitiesForNextPendingManifest())
 	expectError(t, sql.ErrNoRows.Error(), j.CheckVulnerabilitiesForNextManifest())
```

Comment:
The tests should cover that both job loops eventually enter `sql.ErrNoRows`. For example, in this case, we expect:

- that `CheckVulnerabilitiesForNextManifest` returns `sql.ErrNoRows` right away
- that `CheckVulnerabilitiesForNextPendingManifest` succeeds five times and then returns `sql.ErrNoRows` after that

Same for the other instances of the same pattern below. (For some of them, you already changed it in the right way.)

Reply:
I hope I catched all of them